src/ISComm.c

```diff
@@ -81,7 +81,19 @@ unsigned int getBitsAsUInt32(const unsigned char* buffer, unsigned int pos, unsi
 	return bits;
 }
 
-const unsigned int g_validBaudRates[IS_BAUDRATE_COUNT] = { IS_BAUDRATE_3000000, IS_BAUDRATE_921600, IS_BAUDRATE_460800, IS_BAUDRATE_230400, IS_BAUDRATE_115200, IS_BAUDRATE_57600, IS_BAUDRATE_38400, IS_BAUDRATE_19200, IS_BAUDRATE_9600 };
+const unsigned int g_validBaudRates[IS_BAUDRATE_COUNT] = { 
+	                        // Actual on uINS:
+	IS_BAUDRATE_18750000,   // 18750000 (uINS ser1 only) 
+	IS_BAUDRATE_9375000,    // 9375000
+	IS_BAUDRATE_3125000,    // 3125000
```

Comment:
Did we ever support 3000000 baud, or was it always 3125000?

Reply:
Yes, it was always 3125000.  All baudrates were tested on the oscope and look good.